loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
```

Comment:
Agreed, src/dest isn't that clear either, since this isn't a move or copy.  +1 for using source and link_name